qiskit_ibm_provider/ibm_backend_service.py

```diff
@@ -150,7 +150,7 @@ def backends(
                     IBMProvider.backends(open_pulse=True)
 
                 For the full list of backend attributes, see the `IBMBackend` class documentation
-                <https://qiskit.org/documentation/apidoc/providers_models.html>
+                <https://docs.quantum.ibm.com/api/qiskit/providers_models>
```

Comment:
There is no `IBMBackend` class in the Qiskit docs. I'm not sure if this should link to something else?

Reply:
Link looks good